app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -182,6 +183,7 @@ class GithubWebhookSubscription extends SubscriptionHandler {
         break;
       case 'labeled':
         log.info('$crumb: PR labels = [${pr.labels?.map((label) => '"${label.name}"').join(', ')}]');
+        await _processLabels(pullRequestEvent.pullRequest!);
```

Comment:
the pull_request "labeled" event will have "emergency" as the `pullRequestEvent.label.name` field. So you could skip "give me the whole pull request"

Reply:
Yep. `pullRequestEvent.pullRequest.labels` also has the label, so I rewrote it to just use the PR object we're already passing.